gateway/src/apicast/proxy.lua

```diff
@@ -306,55 +310,28 @@ local function response_codes_data()
   return params
 end
 
-local function handle_timer_post_action(_, self, semaphore, cached_key, backend, ...)
-  local res = util.timer('backend post_action', backend.authrep, backend, ...)
-
-  if not using_sync_post_action then
-    semaphore:post(1)
-  end
+local http_backend = {
```

Comment:
Why is this needed?

Reply:
We can't use the `ngx` backend in timers because it uses subrequests. It can use the `ngx` backend only in `access`/ `rewrite`/ `content` phases.

Reply:
:+1: